SU2_CFD/src/output_structure.cpp

```diff
@@ -8593,21 +8598,25 @@ void COutput::SetMesh_Files(CGeometry **geometry, CConfig **config, unsigned sho
       
       /*--- Write a .su2 ASCII file ---*/
 
-      if (rank == MASTER_NODE) cout <<"Writing .su2 file." << endl;
-
-      if (su2_file) SetSU2_MeshASCII(config[iZone], geometry[iZone], iZone, output_file);
-      
-      /*--- Write an stl surface file ---*/
-
-      if (rank == MASTER_NODE) cout <<"Writing .stl surface file." << endl;
-      
-      if (su2_file) SetSTL_MeshASCII(config[iZone], geometry[iZone]);
-      
-      /*--- Write a binary file with the grid coordinates alone. ---*/
-      
-      if (Wrt_Crd) {
-        cout <<"Writing .dat binary coordinates file." << endl;
-        WriteCoordinates_Binary(config[iZone], geometry[iZone], iZone);
+      if (su2_file) {
+        
+        if (rank == MASTER_NODE) cout <<"Writing .su2 file." << endl;
+        
+        SetSU2_MeshASCII(config[iZone], geometry[iZone], iZone, output_file);
+        
+        /*--- Write an stl surface file ---*/
+        
+        if (rank == MASTER_NODE) cout <<"Writing .stl surface file." << endl;
+        
+        SetSTL_MeshASCII(config[iZone], geometry[iZone]);
+        
+        /*--- Write a binary file with the grid coordinates alone. ---*/
+        
+        if (Wrt_Crd) {
```

Comment:
Could you please move the "if (Wrt_Crd) { }" conditional outside of the "if (su2_file) { }" conditional?

Reply:
Thanks for the review! Done!